youtube_dlc/downloader/youtube_live_chat.py

```diff
@@ -27,9 +28,12 @@ def real_download(self, filename, info_dict):
             'total_frags': None,
         }
 
-        def dl_fragment(url):
-            headers = info_dict.get('http_headers', {})
-            return self._download_fragment(ctx, url, info_dict, headers)
+        def dl_fragment(url, data=None, headers=None):
+            http_headers = info_dict.get('http_headers', {})
+            if headers:
+                http_headers = copy.deepcopy(http_headers)
+                http_headers.update(headers)
```

Comment:
Since this is only being updated at toplevel, why use deepcopy? No need to change it, I'm just curious

Reply:
I assumed `info_dict` was supposed to be immutable unlike `ctx`. Just an assumption, so if it makes the code more clear, I can remove the copying

Reply:
No, this is fine. I was just wondering why u used `copy.deepcopy` instead of `http_headers.copy()`

Reply:
Oh, you're right. I'll remove the import and use `.copy` directly